app/models/lti_configuration.rb

```diff
@@ -3,6 +3,14 @@
 class LtiConfiguration < ApplicationRecord
   belongs_to :organization
 
+  enum lms_type: {
+    canvas: 1,
+    blackboard: 2,
+    brightspace: 3,
+    moodle: 4,
+    other: 5
```

Comment:
I know @BenEmdon mentioned this, but what if we put `:other` as 0 (since enums can start at 0) but keep it at the end of the hash? That way we can add a 5th lms and it can be number 5, etc. I also kind of like explicitly keeping `other: 0` because 0 seems more of a "special" case than the other natural numbers.

Reply:
omg love the change to strings didn't know that was even possible ✨